MembraneRTC/Sources/MembraneRTC/Events/Event.swift

```diff
@@ -298,6 +298,17 @@ struct OfferDataEvent: ReceivableEvent, Codable {
         let iceTransportPolicy: String
         let integratedTurnServers: [TurnServer]
         let tracksTypes: [String: Int]
+        
+        init(from decoder: Decoder) throws {
+            let container = try decoder.container(keyedBy: CodingKeys.self)
+            self.integratedTurnServers = try container.decode([TurnServer].self, forKey: .integratedTurnServers)
+            self.tracksTypes = try container.decode([String: Int].self, forKey: .tracksTypes)
+            if container.contains(.iceTransportPolicy) {
+                self.iceTransportPolicy = try container.decode(String.self, forKey: .iceTransportPolicy)
+            } else {
+                self.iceTransportPolicy = "all"
```

Comment:
`iceTransportPolicy` is no longer sent by the server, let's just delete and just set to `relay` in `setTurnServers`  like on Android: https://github.com/membraneframework/membrane-webrtc-android/pull/2#discussion_r901465403

Reply:
@pawelgil ^

Reply:
@graszka22 this change is now in place, please review